frontend/src/components/Widgets/CustomEntitiesDropdown.tsx

```diff
@@ -185,58 +185,122 @@ const styles = {
+                <CustomInputChip sx={styles.chip} entity={item} />
+              ) : (
+                <Chip
+                  sx={{
+                    ...styles.chip,
+                    ...extraChipStyle
+                  }}
+                  variant={"outlined"}
+                  onClick={(e) => {
+                    e.stopPropagation();
+                  }}
+                  deleteIcon={<ClearOutlinedIcon />}
+                  onDelete={!disabled ? (e) => {
+                    e.stopPropagation();
+                    handleChipRemove(item);
+                  } : undefined}
+                  label={item?.label}
+                />
+              )}
+            </Tooltip>
+          ))
+      ) : null}
+      {!disabled && selectedOptions.length > chipsNumber && (
+        <span style={{ marginRight: ".5rem" }}>
+          +{selectedOptions.length - chipsNumber}
+        </span>
+      )}
+    </Box>
+  );
+};
 export default function CustomEntitiesDropdown({
-                                                 value,
-                                                 id,
-                                                 options: {
-                                                   isFormDisabled = () => false,
-                                                   errors,
-                                                   searchPlaceholder,
-                                                   noResultReason,
-                                                   disabledReason,
-                                                   onSearch,
-                                                   onUpdate,
-                                                   mapValueToOption,
-                                                   CustomHeader = null,
-                                                   CustomBody = null,
-                                                   CustomFooter = null,
-                                                   header = {},
-                                                   CustomInputChip = null,
-                                                   placeholder,
-                                                   label,
-                                                   chipsNumber = 2,
-                                                   postProcessOptions = false,
-                                                   refreshStatement,
-                                                   statement,
-                                                   fieldName = "",
-                                                   getPreLevelSelectedValues,
-                                                   areConnectionsExplicit,
-                                                   minWidth = ''
-                                                 },
-                                               }: any) {
+     value,
+     id,
+     placeholder: plcholder,
+     options: {
+       isFormDisabled = () => false,
```

Comment:
question (if-minor) Why do we have isFormDisabled and disabled props? Don't they represent the same?